### DISCUSSION TITLE
Testing

### DISCUSSION THREAD

**Discussion by robert43** [ngageoint/mgrs-ios#1] [Q&A] [answered]
How do I test and use on my iPhone?

**Comment by bosborn** [comment, marked as answer]
You can [build and run](https://github.com/ngageoint/mgrs-ios/tree/master/app#build) the [MGRS app](https://github.com/ngageoint/mgrs-ios/tree/master/app) onto your phone using CocoaPods and XCode, or [include the library](https://github.com/ngageoint/mgrs-ios#import) in your own app. If you are wanting something from the App Store, [MapCache](https://apps.apple.com/us/app/mapcache-by-nga/id1477252454) includes the MGRS grid from this library.
